### ISSUE TITLE
Opensubtitles.com not working

### ISSUE THREAD

**Issue by lodesmets** [morpheus65535/bazarr#1572]
Opensubtitles.com not working/ Job "Search for wanted Series Subtitles (trigger: interval[3:00:00], next run at: 2021-10-06 20:53:37 UTC)" raised an exception

**To Reproduce**
Search for subtitles when using opensubtitles.com

Log of the exception
`
Traceback (most recent call last):
File "/app/bazarr/bin/bazarr/../libs/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/bazarr/bin/bazarr/get_subtitle.py", line 1191, in wanted_search_missing_subtitles_series
    wanted_download_subtitles(episode[\'sonarrEpisodeId\'])
  File "/app/bazarr/bin/bazarr/get_subtitle.py", line 1042, in wanted_download_subtitles
    result = download_subtitle(path_mappings.path_replace(episode[\'path\']),
  File "/app/bazarr/bin/bazarr/get_subtitle.py", line 154, in download_subtitle
    downloaded_subtitles = download_best_subtitles({video}, language_set, int(min_score), hi,
  File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/core.py", line 819, in download_best_subtitles
    subtitles = pool.download_best_subtitles(pool.list_subtitles(video, languages - video.subtitle_languages),
  File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/core.py", line 419, in download_best_subtitles
    if self.download_subtitle(subtitle):
  File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/core.py", line 312, in download_subtitle
    if not subtitle.is_valid():
  File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/subtitle.py", line 269, in is_valid
    text = self.text
  File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/subtitle.py", line 86, in text
    if not self.content:
AttributeError: \'OpenSubtitlesComSubtitle\' object has no attribute \'content\
`

**Comment by lodesmets** [comment]
[bazarr (1).log](https://github.com/morpheus65535/bazarr/files/7309271/bazarr.1.log)

Sorry, I thought I uploaded my logs. 

Sadly I didn't have debug mode enabled when I got the exception, And when I search using opensubtitles.com now, I get a different error. So I think it has something to do with opensubtitles.com and not bazarr.

It happens when I search for anything (not specific content). But in the log file I'm searching for :
"title": "Star Wars Rebels",
    "season": 4,
    "episode": 1,
    "screen_size": "720p",
    "source": "Blu-ray",
    "video_codec": "H.264",
    "release_group": "YELLOWBiRD",
    "container": "mkv",
    "type": "episode"

I'm using Bazarr v0.9.9.
